models/User.js

```diff
@@ -3,6 +3,35 @@ var crypto = require('crypto');
 var mongoose = require('mongoose');
 var Schema = mongoose.Schema;
 
+var knex = require('knex')({
+  client: 'pq',
+  connection: {
+    host : '127.0.0.1',
+    user : 'postgres',
+    password : 'postgres',
+    database : 'myapp_test'
+  }
+});
+
+knex.schema.withSchema('public').createTable('users', function (table) {

```

Comment:
Ah I was only looking at knex, have not looked at that one yet... I'll check it out tonight